### PR TITLE
Remove the use and testing of legacy groupdoc/userdoc/orderdoc docume…

### PR DESCRIPTION
**Pull request by baldersheim**
…nt ids.

@bjorncs or @toregge or @geirst PR
This is just a rewrite of tests. No semantic changes.

### CODE REVIEW

**Comment by geirst** [comment]
I think @vekterli has most context on this one

**Comment by baldersheim** [comment]
@geirst Yes, and he will get the good stuff in later commits. This was just to spread the 6k diff in smaller parts.
This is mostly avoid using userdoc/groupdoc/orderdoc and use id: instead in tests.
Should have been done a long time ago.

**Comment by baldersheim** [comment]
AS multiple upcoming PRs depend on this one I will just merge it.

File: docproc/src/test/java/com/yahoo/docproc/jdisc/DocumentProcessingHandlerTransformingMessagesTestCase.java [hunk 1]
```diff
@@ -202,14 +212,8 @@ public Progress process(Processing processing) {
                     //nada
                 } else if ("doc:nodocstatus:update:to:nothing".equals(id)) {
                     it.remove();
-                } else if ("userdoc:12345:6789:multiop:nodocstatus:keep:this".equals(id)) {
+                } else if ("id:12345:6789:multiop:nodocstatus:keep:this".equals(id)) {
```
Comment (vekterli) [inline docproc/src/test/java/com/yahoo/docproc/jdisc/DocumentProcessingHandlerTransformingMessagesTestCase.java]: Should this have been `n=6789`? Doesn't seem to have broken anything, so maybe not tested at all...

**Review by vekterli — COMMENTED** [review]
:+1:

File: docproc/src/test/java/com/yahoo/docproc/jdisc/DocumentProcessingHandlerTransformingMessagesTestCase.java (hunk 1 above)
Comment (baldersheim) [reply docproc/src/test/java/com/yahoo/docproc/jdisc/DocumentProcessingHandlerTransformingMessagesTestCase.java]: Good point, will check.

**Pull request merged**